src/base/bittorrent/ltqbitarray.cpp

```diff
@@ -52,21 +61,10 @@ namespace BitTorrent::LT
         const char *bitsData = bits.data();
         const int dataLength = (bits.size() + 7) / 8;
 
-        if (dataLength <= STACK_ALLOC_SIZE)
-        {
-            // fast path for small bitfields
-            char tmp[STACK_ALLOC_SIZE];  // uninitialized for faster allocation
-            for (int i = 0; i < dataLength; ++i)
-                tmp[i] = reverseByte(bitsData[i]);
-
-            return QBitArray::fromBits(tmp, bits.size());
-        }
-
-        // slow path for big bitfields
-        auto tmp = std::make_unique<char []>(dataLength);
+        QVarLengthArray<char, STACK_ALLOC_SIZE> tmp(dataLength);
```

Comment:
It can't. Compiler error:
```
qBittorrent/src/base/bittorrent/ltqbitarray.cpp:64:54: error: narrowing conversion of ‘(int)dataLength’ from ‘int’ to ‘char’ [-Wnarrowing]
   64 |         QVarLengthArray<char, STACK_ALLOC_SIZE> tmp {dataLength};
      |                                                      ^~~~~~~~~~
```